drivers/net/loopback.c

```diff
@@ -60,6 +60,15 @@ static void loopback_init(struct net_if *iface)
 	}
 }
 
+#ifdef CONFIG_NET_LOOPBACK_PACKET_DROP
+static float loopback_packet_drop_ratio = 0.0f;
+static float loopback_packet_drop_state = 0.0f;
+
+void loopback_set_packet_drop_ratio(float ratio) {
```

Comment:
The API function should be exposed in a header file. I see there's no `net/loopback.h` yet, I think it should be added.